observability/monitoring/configuring-core-platform-monitoring/configuring-alerts-and-notifications.adoc

```diff
@@ -17,7 +17,7 @@ include::modules/monitoring-disabling-the-local-alertmanager.adoc[leveloffset=+2
 [role="_additional-resources"]
 .Additional resources
 
-* link:https://prometheus.io/docs/alerting/latest/alertmanager/[Alertmanager] (Prometheus documentation)
+* link:https://prometheus.io/docs/alerting/latest/alertmanager/[Alertmanager documentation]
```

Comment:
I think the following is more accurate:

> [Alertmanager] (Prometheus documentation)

Reply:
Consider adding the link to the content in the description box of the PR. https://94837--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/monitoring/configuring-core-platform-monitoring/configuring-alerts-and-notifications